Update OOO Shivering Isles

Sabre142 opened the pull request:
#112
Not sure if NpcFacesForceFullImport is appropriate or if NPC.Eyes, NPC.FaceGen and NPC.Hair is better to use.

Sabre142 committed:
Update OOO Shivering Isles

* Add tag suggestions and removals
 - Add Actors.RecordFlags, modifies actor record flags in 0001A117.
 - Add C.Name, modifies cell names in 0001230D, 00012D3E, 00010353, 00010A68.
 - Add C.Owner, modifies cell ownership in 00012F55.
 - Add EffectStats, modifies Assoc. Weapon in 00094008.
 - Add EnchantmentStats, modifies (ENCH) Enchantment / Object Effect stats in 0005F5E2, 00069677, 0007910F, 000793A0, 0007E09E.
 - Add Invent.Change, changes item count in 0001A354.
 - Add NpcFacesForceFullImport, modifies NPC eyes, faces and hair in 0001559D, 0001207E, 00011F9F, 00011FA1, 00012083.
 - Add R.Eyes, adds eyes to races in 0001208E, 0001208F, 0005308E.
 - Add R.Relations.Add, adds race-faction relationships in 0001208E, 0001208F.
 - Add Sound, replaces sounds in 0004BB77, 0006D63C, 0008DC04, 00016ECC.
 - Add Text, modifies DESC - Description field in 000181E1, 0001F3C6, 0001F4C2, 00043523, 00043981.
 - Remove R.Body-M, does not modify male body mesh/texture definitions.
 - Remove Relations.Change, does not change faction relationship modifiers.

* Add cleaning data
 - Version 2.11: Pre-QAC: 0x252B8019, Post-QAC: 0xC5ED29BA.

MacSplody commented on masterlist.yaml:
```diff
@@ -11295,13 +11295,30 @@ plugins:
       - name: -R.Head
         condition: 'file("Oblivion_Character_Overhaul.esp")'
+      - R.Relations.Add
+      - -Relations
+      - Relations.Add
```
For removal of tags, you can ignore the deprecated ones.
But you can remove part of the deprecated tags behavior using the new tags.

```suggestion
      - Relations.Add
      - -Relations.Change
```

MacSplody commented:
> Not sure if NpcFacesForceFullImport is appropriate or if NPC.Eyes, NPC.FaceGen and NPC.Hair is better to use.

NpcFacesForceFullImport would be appropriate if it needs to overwrite later loading plugins (with NPC.Eyes, NPC.FaceGen and NPC.Hair tags).
If you're not sure then use the separate tags.

MacSplody commented on masterlist.yaml:
```diff
@@ -11295,13 +11295,30 @@ plugins:
     url: [ 'https://www.nexusmods.com/oblivion/mods/46508/' ]
     tag:
+      - Actors.RecordFlags
+      - -Body-M
```
You can replace `-Body-M` with `-R.Body-M`.

Sabre142 committed:
Update OOO Shivering Isles

Co-authored-by: MacSplody <1944639+MacSplody@users.noreply.github.com>

Sabre142 committed:
Update OOO Shivering Isles

- Replace NpcFacesForceFullImport tag with NPC.Eyes, NPC.FaceGen, NPC.Hair.
- Replace -Body-M with -R.Body-M.

MacSplody commented:
Thanks.
